### ISSUE TITLE
Extension not doing anything

### ISSUE THREAD

**Issue by StridingDragon** [lpubsppop01/vscode-auto-timestamp#3]
This extension does not work at all, but perhaps it's just because there is zero explanation or documentation on how to use it.

Using it in a C file in VSCode 1.34.0 on OSX 10.14.5 and try to embed time stamps, but nothing happens. First I tried this…

```
// Created: 
// Last modified: 
```

Then I tried 
```
// Created: <created date and time>
// Last modified: <now date and time>
```
What am I missing?

**Comment by lpubsppop01** [comment]
Hi, @StridingDragon.
Thanks for your feedback.

I checked using the old mac (EI Capitain) and a C file that contains the text posted by you at the beginning, but it worked as normal.
It was a combination that I did not check, but it does not seem to not work due to the environment.

The settings for this extension include file name pattern (default .*) and target number of lines (default 5 lines from head).
If you haven't changed the settings, the line count setting may be too small for your file.
